design_drafts/joint_interfaces_and_transmissions.md

```diff
@@ -35,49 +35,57 @@ and ties in with the use of the [flexible joint states message](https://github.c
 ## Proposal
 
 ### urdf
-Over the years we didn't receive any feedback about the URDF notation being good/bad which means it's not currently a pain point for anyone at the moment, this design will not propose any changes to that.
-Let's stick with the notation:
+
 ```
 <transmission name="extended_simple_trans">
     <type>transmission_interface/ExtendedSimpleTransmission</type>
     <joint name="joint1">
-      <offset>0.0</offset>
-      <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
-      <hardwareInterface>hardware_interface/VelocityJointInterface</hardwareInterface>
-      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
-      <hardwareInterface>awesome_interface/FooJointInterface</hardwareInterface>
-      <role></role>
+      <actuator>
```

Comment:
This updated version is a little too ad-hoc.

You gotta have joints and actuators. With names and all that which you forgot to add. Also, I added an example of simpletransmission here as this one only defines a 1-to-1 relationship, we can highlight additional sensors but let's not have multiple joints and actuators in it.

Here's a ref you can use to establish a more complex example: https://github.com/pal-robotics/tiago_robot/blob/kinetic-devel/tiago_description/urdf/arm/wrist.transmission.xacro#L28